test/unit/authorizer_test.rb

```diff
@@ -112,7 +112,7 @@ def setup
     domain1     = FactoryGirl.create(:domain)
     domain2     = FactoryGirl.create(:domain, :name => 'a-domain.to-be-found.com')
     domain3     = FactoryGirl.create(:domain, :name => 'another-domain.to-be-found.com')
-    domain4     = FactoryGirl.create(:domain, :name => 'be_editable.to-be-found.com')
+    domain4     = FactoryGirl.create(:domain, :name => 'be-editable.to-be-found.com')
```

Comment:
As far as I can tell (after browsing through quite a few RFCs), underscores in domain names are only used for [SRV](https://tools.ietf.org/html/rfc2782) (A protocol for locating which server supplies a specific service) specifically so they won't conflict with actual hostnames. 
I don't think that in our case we should allow underscores as domain names are used primarily for assigning to hosts & registering to DNS.